frontend/packages/shipwright-plugin/src/api.ts

```diff
@@ -1,10 +1,11 @@
 import { k8sCreateResource } from '@console/dynamic-plugin-sdk/src/utils/k8s';
-import { BUILDRUN_TO_BUILD_REFERENCE_LABEL } from './const';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { BUILDRUN_TO_BUILD_REFERENCE_LABEL, BUILDRUN_TO_RESOURCE_MAP_LABEL } from './const';
 import { BuildRunModel } from './models';
 import { Build, BuildRun } from './types';
 
 /** Create a new BuildRun for a given Build to start it. */
-export const startBuild = async (build: Build): Promise<BuildRun> => {
+export const startBuild = async (build: Build, resource?: K8sResourceKind): Promise<BuildRun> => {
```

Comment:
Can you add a new test case in packages/shipwright-plugin/src/__tests__/api.spec.ts?

Reply:
This additional argument is not required now since we are attaching the label with the Build too. 
Therefore new test case is not required as no change in the function definition.